### PR TITLE
[IMP] payment(_paypal): add missing info

### PR DESCRIPTION
**Pull request by anko-odoo**
KBC has been added to Mollie.
Cards support refund.
Refund is now checked based on payment method refund support, not brand. "Cleared" mapping has been added to done state in PayPal in order to properly support echeck.

opw-3880431

### CODE REVIEW

File: addons/payment_paypal/const.py [hunk 1]
```diff
@@ -41,6 +41,6 @@
 # See https://developer.paypal.com/docs/api-basics/notifications/ipn/IPNandPDTVariables/
```
Comment (vchu-odoo) [inline addons/payment_paypal/const.py:41]: There is no info about 'Cleared' status in the link. Maybe you could at least add the doc link about 'Cleared'  status to the dev notes of the task?

**Review by vchu-odoo — APPROVED** [review]
Hello,
nice work!
LGTM :+1: 

Only some small nitpickings.

File: addons/payment_mollie/const.py [hunk 2]
```diff
@@ -66,4 +66,5 @@
     'bank_transfer': 'banktransfer',
     'p24': 'przelewy24',
     'sepa_direct_debit': 'directdebit',
+    'kbc_cbc': 'kbc',
```
Comment (AntoineVDV) [inline addons/payment_mollie/const.py:69]: Alphabetical order to ease maintenance.

File: addons/payment/data/payment_provider_data.xml [hunk 3]
```diff
@@ -271,6 +271,7 @@
                          ref('payment.payment_method_paysafecard'),
                          ref('payment.payment_method_p24'),
                          ref('payment.payment_method_sofort'),
+                         ref('payment.payment_method_kbc_cbc'),
```
Comment (AntoineVDV) [inline addons/payment/data/payment_provider_data.xml:274]: same

File: addons/payment_paypal/const.py (hunk 1 above)
Comment (AntoineVDV) [reply addons/payment_paypal/const.py:41]: I'm not sure we even have documentation about this; at least it should be a comment.

**Review by AntoineVDV — CHANGES_REQUESTED** [review]
Don't forget to apply @vchu-odoo's review comments too :)

**Comment by AntoineVDV** [comment]
Removed rd-accounting review request since it's a small payment-only fix.

**Comment by anko-odoo** [comment]
@AntoineVDV

**Review by AntoineVDV — APPROVED** [review]
@robodoo r+

**Comment by anko-odoo** [comment]
@robodoo r+

**Pull request closed without merging**